### ISSUE TITLE
Using QCOW2 instead of VMDK

### ISSUE THREAD

**Issue by NathanDotTo** [CiscoDevNet/opendaylight-setup#2]
Hi Giles

The scripts that you started with use a VMDK, which I know that you obtain by means not available to mere mortals. The only way that someone outside of Cisco can obtain a VMDK is to extract it from the OVA. by untar'ing the OVA.

Trying that VMDK, though, results in this error:

./XRe 
qemu-system-x86_64: -hda ./XRe.vmdk: VMDK version 3 must be read only

I seem to have more success with using the distributed QCOW2 image instead of the VMDK. 

Regards

Nathan

**Comment by gilesheron** [comment]
yeah - I'll probably test it with the QCOW2 today.   Do you have a link for it?
